model/configuration.py

```diff
@@ -236,6 +257,45 @@ def cache_key(self):
         # bit and by_cache_key() not as much.
         return self.id
 
+    @classmethod
+    def get_integrations_of_goal(cls, _db, goal):
```

Comment:
For consistency with other methods this should be called `get_integrations_for_goal`.